__tests__/parameter-description-warning/openapi.yaml

```diff
@@ -0,0 +1,48 @@
+openapi: 3.0.2
+info:
+  title: Example OpenAPI 3 definition. Valid.
+  version: 1.0
+
+components:
+  parameters:
+    offsetParam:
+      in: query
+      name: offset
+      required: false
+      schema:
+        type: integer
+        minimum: 0
+      #description: The number of items to skip before starting to collect the result set.
```

Comment:
I'd rather clean up the example and leave a single parameter without description to make the snapshot cleaner.